vendor/src/github.com/docker/libnetwork/controller.go

```diff
@@ -1003,9 +1003,9 @@ func SandboxKeyWalker(out *Sandbox, key string) SandboxWalker {
 func (c *controller) loadDriver(networkType string) error {
 	// Plugins pkg performs lazy loading of plugins that acts as remote drivers.
 	// As per the design, this Get call will result in remote driver discovery if there is a corresponding plugin available.
-	_, err := plugins.Get(networkType, driverapi.NetworkPluginEndpointType)
+	_, err := pluginstore.LookupWithCapability(networkType, driverapi.NetworkPluginEndpointType)
```

Comment:
Yes. The approach is to try with pluginv2 first. If that fails, fallback to pluginv1. The fallback is controlled by allowV1PluginsFallback, which will be true for 1.13.